### DISCUSSION TITLE
How to split test cases in pytest

### DISCUSSION THREAD

**Discussion by ramachandargadde** [pytest-dev/pytest#10048] [Q&A] [answered]
I got around 2500+ test cases

I need to split the test cases so that I can run the tests on different runners. 
for us using the markers isn't the best option.

is there any other way like collecting the first 500 tests and 500-1000 tests like ways, so that we can trigger tests on multiple runners.
using some simple tags are any other ways?

**Comment by RonnyPfannschmidt** [comment, marked as answer]
The pytest-split plugin may be a option

**Reply by ramachandargadde** [reply]
Thanks, a lot Ronny
